<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>431</width>
      <height>353</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="labelCity">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>12</y>
       <width>114</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>xx市xx县</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTemperature">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>51</y>
       <width>132</width>
       <height>140</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string>XX</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelIcon">
     <property name="geometry">
      <rect>
       <x>219</x>
       <y>51</y>
       <width>201</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>72</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img/img.qrc">:/weather.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="labelHigh">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>258</y>
       <width>115</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>高温 xx.x℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLow">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>258</y>
       <width>115</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>低温 xx.x℃</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>288</y>
       <width>329</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最后更新于：20xx-xx-xx 23:59:59</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelFengSu">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>318</y>
       <width>133</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>风速：xx风x级</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSheshidu">
     <property name="geometry">
      <rect>
       <x>152</x>
       <y>51</y>
       <width>60</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>℃</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
